public/sass/pages/_dashboard.scss

```
@@ -102,11 +102,22 @@ div.flot-text {
   width: 100%;
   flex-grow: 1;
   height: calc(100% - #{$panel-header-height});
+  overflow: hidden;
 
   &--no-padding {
     padding: 0;
   }
+
+  &--scrollable-y {
+    overflow: hidden auto;
+    margin-bottom: $panel-padding;
+  }
+
+  &--no-padding.panel-content--scrollable-y {
```

Note: can I write this in some better way using sass?

---

```suggestion
  &--no-padding#{&}--scrollable-y {
```

might do the trick :)